app/victoria-metrics/main_test.go

```diff
@@ -416,6 +438,36 @@ func removeIfFoundQueryData(r QueryDataResult, contains []QueryDataResult) []Que
 	return contains
 }
 
+func checkQueryRangeResult(t *testing.T, got, want QueryRange, query, issue string) {
+	t.Helper()
+	if got.Status != want.Status {
+		t.Fatalf("query %s. Result Query Range status mismatch %q - %q. %s", query, want.Status, got.Status, issue)
```

Comment:
I'd prefer `checkQueryRangeResult` to return an err, so caller could wrap err msg with context:
```
if err:=checkQueryRangeResult(queryResult, test.ResultQueryRange, q, test.Issue); err != nil {
  t.Fatalf("query %q failed: %s", err) 
}

```